include/llbuild/Basic/ExecutionQueue.h

```diff
@@ -77,6 +77,11 @@ namespace llbuild {
       void execute(QueueJobContext* context) { work(context); }
     };
 
+    enum QueueJobPriority {
```

Comment:
Just a suggestion, for improved code readability it would be better to make that an `enum class` so at use site you'd use `basic::QueueJobPriority::High` instead of `basic::High`.
`Hight` and `Normal` as names do not have enough context on their own.

Reply:
Ah yes, my modern c++ is rusty.  So much swift these days.